config/locales/server.en.yml

```diff
@@ -4506,7 +4506,7 @@ en:
 
       ## [Powered by You](#power)
 
-      This site is operated by your [friendly local staff](%{base_path}/about) and *you*, the community. If you have any further questions about how things should work here, open a new topic in %{feedback_category} and let’s discuss! If there’s a critical or urgent issue that can’t be handled by a meta topic or flag, contact us via the [staff page](%{base_path}/about).
+      This site is operated by your [friendly local staff](%{base_path}/about) and *you*, the community. If you have any further questions about how things should work here, open a new topic in %{feedback_category_hashtag} and let’s discuss! If there’s a critical or urgent issue that can’t be handled by a meta topic or flag, contact us via the [staff page](%{base_path}/about).
```

Comment:
Can we keep the original `%{feedback_category}` placeholder? This hasn't been pushed to Crowdin yet, so have the possibility to revert the rename. Otherwise we will lose all translations for this very long string again. Our translators just finished working on that, it would be a shame if they need to redo it again. :(